test/integration.js

```diff
@@ -258,6 +258,38 @@ tape('POST /channel/{id}/{validator-messages}: wrong signature', function(t) {
 	.catch(err => t.fail(err))
 })
 
+tape('POST /channel/{id}/{validator-messages}: wrong root hash', function(t) {
```

Comment:
can you change that to "wrong (deceptive) root hash"

Reply:
okay will do

Reply:
okay